tests/integration/serverConnections.integration.test.ts

```diff
@@ -127,6 +127,7 @@ describe('Mixer connections', () => {
     describe('Connections that should be denied', () => {
         beforeEach(() => {
             hifiCommunicator = new HiFiCommunicator();
+            jest.setTimeout(30000);
```

Comment:
These timeouts allow time for auto-reconnects to finish up when we're defaulting auto-reconnects to "true"